src/cephadm/cephadm

```diff
@@ -1259,6 +1260,12 @@ def create_daemon_dirs(fsid, daemon_type, daemon_id, uid, gid,
             os.fchmod(f.fileno(), 0o600)
             os.fchown(f.fileno(), uid, gid)
             f.write(keyring)
+    if extra_files:
```

Comment:
hm. can we use a similar scheme to Monitoring?

```python 
data = get_parm(args.config_json)
```

Reply:
I'll take a look :) Thanks